beekeeper-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/generator/utils/CleanupDelayExtractor.java

```diff
@@ -46,11 +46,16 @@ public CleanupDelayExtractor(String propertyKey, String defaultValue) {
   public Duration extractCleanupDelay(ListenerEvent listenerEvent) {
     String tableCleanupDelay = listenerEvent.getTableParameters().get(propertyKey);
     try {
-      return Duration.parse(tableCleanupDelay);
+      Duration value = tableCleanupDelay == null ? defaultValue : Duration.parse(tableCleanupDelay);
+      log
+          .info("Using value '{}' for key {} for table {}.{}.", value, propertyKey, listenerEvent.getDbName(),
+              listenerEvent.getTableName());
+      return value;
     } catch (DateTimeParseException | NullPointerException e) {
-      log.error(
-          "Overridden delay value '{}' for key '{}' cannot be parsed to a Duration for table '{}.{}'. Using default setting.",
-          tableCleanupDelay, propertyKey, listenerEvent.getDbName(), listenerEvent.getTableName());
+      log
+          .warn(
```

Comment:
v picky, but is it the formatter thats splitting `log.info` and `log.warn` onto separate lines? Think it would be nicer on one line

Reply:
Do you mean to put the log at the end? So we only use 1 log print statement?

Reply:
no I mean have `log.warn` instead of 
```
log
   .warn
```

Reply:
I can do it, but my format settings (from HCOM) recommend to put it in 2 lines.